ideascube/mediacenter/templates/mediacenter/index.html

```diff
@@ -21,13 +21,25 @@
         </div>
     {% endif %}
     {% url 'mediacenter:index' as index %}
+    {% include 'search/filter_cloud.html' with action=index %}
     {% include 'search/box.html' with action=index %}
+    {% if not_empty_kinds|length > 1 %}
     <div class="card tinted">
         <h4><span class="theme discover">{% trans "browse" %}</span> {% trans 'by kind' %}</h4>
-        {% for kind, label in view.model.KIND_CHOICES %}
-            <a href="?kind={{ kind }}">{{ label }}</a>
+        {% for kind, label in not_empty_kinds %}
+            <a href="{% add_qs kind=kind %}">{{ label }}</a>
             {% if not forloop.last %}⚫{% endif  %}
         {% endfor %}
     </div>
-    {% tag_cloud url='mediacenter:by_tag' model=view.model %}
+    {% endif %}
+    {% if not_empty_langs|length > 1 %}
```

Comment:
nit: not a big fan a double negated naming (not + empty), I'd go for "available_xxx" instead :)